.env.local

```diff
@@ -16,15 +16,24 @@ export DATA_RENTGEN__UI__API_BROWSER_URL=http://localhost:8000
 export DATA_RENTGEN__SERVER__SESSION__SECRET_KEY=session_secret_key
 
 # Keycloak Auth
-export DATA_RENTGEN__AUTH__KEYCLOAK__SERVER_URL=http://keycloak:8080
+export DATA_RENTGEN__AUTH__KEYCLOAK__SERVER_URL=http://localhost:8080
 export DATA_RENTGEN__AUTH__KEYCLOAK__REALM_NAME=manually_created
 export DATA_RENTGEN__AUTH__KEYCLOAK__CLIENT_ID=manually_created
-export DATA_RENTGEN__AUTH__KEYCLOAK__CLIENT_SECRET=generated_by_keycloak
-export DATA_RENTGEN__AUTH__KEYCLOAK__REDIRECT_URI=http://localhost:8000/auth/callback
+export DATA_RENTGEN__AUTH__KEYCLOAK__CLIENT_SECRET=change_me
+export DATA_RENTGEN__AUTH__KEYCLOAK__REDIRECT_URI=http://localhost:3000/callback
```

Comment:
Maybe `/auth/callback`?

Reply:
This is route on frontend client.

Reply:
I understand that, but the URL is not descriptive now.

Reply:
OK